perf/istio-install/setup_istio.sh

```diff
@@ -108,18 +108,94 @@ function install_extras() {
   # Redeploy, this time with the Prometheus resource created
   helm template --set domain="${domain}" "${WD}/base" | kubectl apply -f -
   # Also deploy relevant ServiceMonitors
-  "${release}/bin/istioctl" manifest generate --set profile=empty --set addonComponents.prometheusOperator.enabled=true -d "${release}/manifests" | kubectl apply -f -
+  if [[ -f "${release}/samples/addons/extras/prometheus-operator.yaml" ]];then
+     kubectl apply -f "${release}/samples/addons/extras/prometheus-operator.yaml" -n istio-system
+  # for release before 1.7, run below instead
+  else
+    "${release}/bin/istioctl" manifest generate --set profile=empty --set addonComponents.prometheusOperator.enabled=true -d "${release}/manifests" | kubectl apply -f -
+  fi
   # deploy grafana
   kubectl apply -f "${release}/samples/addons/grafana.yaml" -n istio-system
 }
 
+# Install ASM with kpt and istioctl on an existing gke cluster
+# This may not work with older version of ASM before 1.6.
+# You might need to Set ASM_HUB and ASM_TAG env variables accordingly, otherwise default values would be used.
+function install_asm() {
```

Comment:
this does not belong here

Reply:
This is for OSS Istio...

Reply:
I added here because this is extracted from public instruction and try to align usage, do you mean another separate script/branch/repo?

Reply:
Anywhere but `istio` org

Reply:
hmm I think @mandarjog mentioned that it is ok to add to OSS, and actually we have the anthoscli instruction in tools/ now

Reply:
I suggest that we move this into a subdirectory. `vendor/google` any google specific tools code can live there.
There is nothing really proprietary here, but it is good for cleanliness. If a different vendor needs to have their stuff, they can also add it to the vendor directory.

Reply:
Agree I would like to keep this vendor neutral as it'll make the IBM KS instructions harder. There are a few items that are GKE specific that I have to already doc around (once this is all in).

Reply:
ok, moved related scripts under vendor/google